SU-HSDO/suhumsci
# STN-342 : New Traditional Color Pairing

# [STN-342](https://sparkbox.atlassian.net/browse/STN-342) READY FOR REVIEW

## Summary
The work in this PR includes:
1. In the Colorful color pairings, update `Sand` to `Cardinal`.
2. In the Traditional color pairings, add `Blue Jay`.

Reference: [Color Pairing Spreadsheet](https://docs.google.com/spreadsheets/d/1m2Dt4trd2qLZJhsuXidLBtTRovlm4tLwRyG3ic07ANM/edit#gid=0)

_Note:_ The Traditional Cardinal color pairings have been updated to match the values in the [color pairing spreadsheet](https://docs.google.com/spreadsheets/d/1m2Dt4trd2qLZJhsuXidLBtTRovlm4tLwRyG3ic07ANM/edit#gid=0). I connected with Merani to confirm that the spreadsheet is the source of truth before making the changes. `Primary Dark` is the only exception. I did not update it as I recall it was updated last sprint. There is a note in the spreadsheet regarding the value in the codebase.

## Need Review By (Date)
5/15/2020

## Urgency
low

## Steps to Test
1. Run `npm run test` and verify all of our Sass tests continue to pass.
2. The differences between the Traditional themes `Cardinal` and `Blue Jay` color pairings are somewhat subtle. The most notable difference is seen in the hero and carousel components. In the Traditional theme, go to the QA Page. When the `Cardinal` color pairing is implemented, the color of the image overlay is charcoal grey in color. Switch to the `Blue Jay` color pairing by going to `Appearance > Settings > Humsci Traditional > Color Pairing > select `Blue Jay`. Save configuration and go back to the QA Page. The color of the image overlay should be a shade blue.
3. I also recommend testing the application of the `ht-pairing-bluejay` class on a block in the layout builder.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
